src/Response/Response.cpp

```diff
@@ -0,0 +1,94 @@
+#include "Response/Response.hpp"
+
+/*
+** ------------------------------- CONSTRUCTOR --------------------------------
+*/
+
+Response::Response() {}
+
+Response::Response(int statusCode, std::string contentType,
+                   const std::string &body) {
+  _header.protocolVersion = "HTTP/1.1";
+  _header.statusCode = statusCode;
+  setReasonPhrase(statusCode);
+  _header.contentType = contentType;
+  _header.contentLenght = body.length();
+  _body = body;
+}
+
+/*
+** -------------------------------- DESTRUCTOR --------------------------------
+*/
+
+Response::~Response() {}
+
+/*
+** --------------------------------- OVERLOAD ---------------------------------
+*/
+
+Response &Response::operator=(const Response &assign) {
+  if (this != &assign) {
+    _header = assign._header;
+  }
```

Comment:
The copy assignment operator is not copying the *_body* from the right hand side.